### ISSUE TITLE
Add dist type file for packages that are single plain text files.

### ISSUE THREAD

**Issue by disposable-ksa98** [composer/composer#946]
A dist type "file" should be introduced which downloads a plaintext file and keeps it as-is. This is useful if a dependency consists of a single file which is not distributed in an archive. For example jquery is available as a single js file.

------ Original Content ------

Please check out this question, it seems like I'm not the only one having this issue: http://stackoverflow.com/questions/11663559/how-to-install-jquery-with-composer

Basically we need another 'type' for packages (unless it's already there but not in the docs). Currently there is 'zip' with which you can install this for example: http://yui.zenfs.com/releases/yuicompressor/yuicompressor-2.4.7.zip. But how do you install this?: http://code.jquery.com/jquery-1.7.2.js
